$NetBSD: patch-etc_mk-bdic,v 1.1 2013/12/21 03:20:16 makoto Exp $

Make it as a function shape, otherwise just loading may
stall at completion

--- etc/mk-bdic.orig	2013-12-21 12:06:36.000000000 +0900
+++ etc/mk-bdic	2013-12-21 12:02:10.000000000 +0900
@@ -1,4 +1,6 @@
 ; -*-emacs-lisp-*-
+(provide 'mk-bdic)
+(defun mk-bdic ()
 (and (fboundp 'set-language-environment)
      (set-language-environment "Japanese"))
 (setq load-path (cons "../lisp" load-path))
@@ -16,3 +18,4 @@
   (tcode-save-dictionaries)
   (message "making bushu dictionary...done"))
 (kill-emacs)
+)
